zebra-chain/src/transaction/joinsplit.rs

```diff
@@ -21,13 +26,9 @@ pub struct JoinSplit<P: ZkSnarkProof> {
     /// XXX refine type
     pub anchor: [u8; 32],
```

Comment:
non-blocking for this PR, but: do you know what type `anchor` should be?

Reply:
The root of the Sapling note commitment merkle tree, which uses Pedersen hashes at each layer except the leaves, so this is a `jubjub::Fq` (as itself or as pure bytes? Do we want a type for that? I have a dumb `SaplingNoteTreeRootHash` that's been floating around for ages as a placeholder that currently wraps 32 bytes.

Reply:
I think we want to try to keep things in the transaction structures byte-wrappers as much as possible, but a `SaplingNoteTreeRootHash` byte wrapper sounds good. We can clean up the naming later.